docs/multicluster/user-guide.md

```diff
@@ -559,6 +559,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Antrea Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster allows traffic between Pods in different clusters. If you want to enable this feature,
+the Pod CIDRs must be set in the ConfigMap `antrea-mc-controller-config` like below. You can edit the config in
```

Comment:
Add a line to highlight no CIDR overlapping. eg: `Pod CIDRs must not overlap among clusters`

Reply:
Added a Notice with bold.